cypress/e2e/awx/resources/inventoriesConstructed.cy.ts

```diff
@@ -22,36 +19,34 @@ describe('Constructed Inventories CRUD Tests', () => {
       cy.createAwxInstanceGroup().then((ig) => {
         instanceGroup = ig;
       });
+      cy.createInventoryHost(organization, 'constructed').then((result) => {
+        const { inventory: inv } = result;
+        newInventory = inv;
+      });
     });
   });
 
   beforeEach(() => {
-    inventoriesList = [];
-    invNames = [];
-    for (let i = 0; i < invToCreate; i++) {
-      cy.createAwxInventory(organization).then((inv) => {
-        inventoriesList.push(inv);
-      });
-    }
+    cy.createAwxInventory(organization).then((inv) => {
```

Comment:
I think we are creating this Inventory to prepopulate the input inventories field on the constructed inventory form.  If thats true, lets create this inventory 1 time in the `before`, and then delete it 1 time in the `after` hooks.